test/libsolidity/syntaxTests/array/calldata_resize.sol

```diff
@@ -1,7 +0,0 @@
-contract C {
-    function f (uint256[] calldata x) external pure {
-        x.length = 42;
-    }
-}
-// ----
-// TypeError: (75-83): Calldata arrays cannot be resized.
```

Comment:
Why do you kill the error messages I just added recently and delete their tests ;-)?